__tests__/app.test.js

```diff
@@ -0,0 +1,23 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => seed(testData));
+

```

Comment:
I'd recommend adding and afterAll here also to close the db connection after your tests run. This will help you to avoid open handles errors.